canvas_exercise/shapes_game/index.js

```diff
@@ -1,36 +1,121 @@
+/* jshint esversion: 6 */
+
```

Comment:
Good job on this.  Your code looks good and all the features seem good when I run the code.